konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/step/bbox/point/BoundingBoxToPointStep.java

```diff
@@ -41,8 +42,17 @@
 public class BoundingBoxToPointStep implements PipelineStep {
     public enum ConversionMethod {TOP_LEFT, TOP_RIGHT, BOTTOM_LEFT, BOTTOM_RIGHT, CENTER}
 
-    private String bboxName;       //If null: just find any BB's
+    @Schema(description = "Name of the bounding boxes key from the previous step. If set to null, it will try to find any bounding box in the incoming data instance.")
+    private String bboxName;
+
+    @Schema(description = "Name of the key of point representation from this step")
```

Comment:
`Name of the point extracted from the input bounding box. If null, the input field name is used`